lib/launch/handler_functions.ex

```diff
@@ -17,53 +25,90 @@ defmodule Membrane.OpenTelemetry.Plugs.Launch.HandlerFunctions do
 
   defp do_start_span(component_type, component_state)
 
-  defp do_start_span(:pipeline, _component_state) do
-    Membrane.OpenTelemetry.start_span(@span_id)
-    Process.put(@pdict_key_span_alive?, true)
+  defp do_start_span(:pipeline, component_state) do
+    # trick to mute dialyzer
```

Comment:
What was the dialyzer's error?

Reply:
Dialyzer generates dumb errors because it sees result of `tracked_pipelines/0` in the compile time